eclass/kernel-2.eclass

```diff
@@ -645,11 +642,7 @@ if [[ ${ETYPE} == sources ]]; then
 			DEBLOB_CHECK_A="deblob-check-${DEBLOB_PV}"
 			DEBLOB_HOMEPAGE="http://www.fsfla.org/svn/fsfla/software/linux-libre/releases/tags"
 			DEBLOB_URI_PATH="${DEBLOB_PV}${K_DEBLOB_TAG}"
-			if ! has "${EAPI:-0}" 0 1 ; then
-				DEBLOB_CHECK_URI="${DEBLOB_HOMEPAGE}/${DEBLOB_URI_PATH}/deblob-check -> ${DEBLOB_CHECK_A}"
-			else
-				DEBLOB_CHECK_URI="mirror://gentoo/${DEBLOB_CHECK_A}"
-			fi
+			DEBLOB_CHECK_URI="mirror://gentoo/${DEBLOB_CHECK_A}"
```

Comment:
I may have done this backwards; the if here is to check for EAPI 0-1 or not EAPI 0-1